src/components/Button/Button.tsx

```diff
@@ -93,7 +93,7 @@ export function Button(props: ButtonProps) {
     <button
       data-testid="button"
       type="button"
-      className={`default-button px-4 py-3 ${classes[variant]} ${customClass}`}
+      className={`default-button flex items-center px-4 ${classes[variant]} ${customClass}`}
```

Comment:
Have you checked if it won't break the icon buttons varition!? 🤔

Reply:
No idea, but I have the feeling it could break. Specially the `min-h-12`... 🤔  Please, double check!

Reply:
All buttons on the wallet and station have a min height of 48px or 12 in tailwind, so I added in the min height to be equal tp the current min height

Reply:
If you say...

Reply:
I mean if your not confident about this we don't have to go forward, the UI works perfect as is... not really a necessary change, it's more about being pixel perfect